packages/mds-logger/redaction.ts

```diff
@@ -1,8 +1,8 @@
 // TODO: This is an inefficient method, and we should be doing this in an external Pino transport (see PLAT-195)
-export const redact = (arg: string | Record<string, unknown> | Error | undefined) => {
+export const redact = (arg: string | Record<string, unknown> | Error | undefined, keys: Set<string>) => {
   if (arg === undefined) return {}
   const res = JSON.stringify(arg instanceof Error ? { error: arg.toString() } : arg, (k, v) =>
-    ['lat', 'lng'].includes(k) ? '[REDACTED]' : v
+    keys.has(k) ? '[REDACTED]' : v
```

Comment:
Weren't we going to do some "fuzzy" matching here ... lowercase everything and remove special characters before comparison? So 'lat', 'Lat', 'LAT', etc. would all get redacted?

Reply:
Ahh, yep we should definitely try to add that.

Reply:
Added